### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFAnnotationsStyling_Screenreader: The options present in the "StickyNote Style" combo box is not announced to the screen reader users.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1490]
Copied from VSO [957351](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/957351)

Environment Details:
Application Name: .NET Core 3.0 (WPF)

Application version: VS 2019 Preview 4.7.03190

Windows Version: Windows10

ScreenReader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link.

Repro Steps:

1) Run exe file for "AnnotationsStyling" from the build app folder provided.
2) Main Window for "AnnotationsStyling" solution will appear.
3) Start screen reader.
4) Navigate to the "StickyNote Style" combo box and navigate through it's options.
5) Listen how are they announced.

Actual:

When screen reader users navigates through the options of the  "StickyNote Style" combo box and the option names are not announced as per it's visual appearance on the screen. 

The same issue is observed using the both the screen readers i.e. Narrator and NVDA.

Expected:

The options of the  "StickyNote Style" combo box should be announced properly to the screen reader users as per it's visual appearance.

User Impact:

If the options of a combo box is not announced correctly to the screen reader users then they might not understand which option should be selected and they will also get confused whether the option which they are selecting is correct or not.

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by rladuca** [comment]
This is an application bug, not a WPF bug.

The `StyleMetaData` class (which comprises the ComboBoxItems) did not have a proper `ToString` override.  If you add an override to simply return the `Key` string, narrator reads the correct values.

Closing.
